src/validator.js

```diff
@@ -272,6 +189,70 @@ export default class Validator {
     return output;
   }
 
+  getAddonMetaData() {
+    return this.xpi.getMetaData()
```

Comment:
Should this logic go into the `XPI` class so the `Validator` class isn't directly loading Parsers? I like the idea of stripping the validator down in terms of the logic in it.